src/workloads/execution/BestBuyInserts.yml

```diff
@@ -0,0 +1,141 @@
+     dataset, those that would be useful for the typical query workloads.
+   - Only a column store index. A column store index isn't expected to realistically replace all of
+     those indexes in production, but it could replace some and knowing it's relative performance
+     cost could help make that decision.
+
+  In order to run this workload, the bestbuy dataset must be present already in the database under
+  test. This is designed to test insert performance in more of a steady-state and hopefully I/O
+  bound mongod.
+
+  The workload consists of the following phases and actors:
+   - Bulk inserting with the deefault indexes. (SamplingLoader used to generate artificial inserts)
+   - Drop all non-_id indexes (RunCommand Actor)
+   - Measure bulk inserting again (SamplingLoader)
+   - Build the column store index (RunCommand Actor)
+   - Measure bulk inserting again (SamplingLoader)
+
+Keywords:
+- columnstore
+- analytics
+- scale
+- insert
+
+Clients:
+  Default:
+    QueryOptions:
+      # Allow for longer duration since index builds may take a while.
+      socketTimeoutMS: 3_600_000  # = 1 hour
+      connectTimeoutMS: 3_600_000
+
+GlobalDefaults:
+  # Defines the database name.
+  Database: &db bestbuy
+
+  # Defines the collection name.
+  Collection: &Collection products
+
+  ColumnStoreIndexSpec: &ColumnStoreIndexSpec
+    key: {$**: "columnstore"}
+    name: "columnstore"
+
+  LoadThreads: &LoadThreads 8
+
+  SamplingLoaderPhaseConfig: &SamplingLoaderPhaseConfig
+    Collection: *Collection
+    Database: *db
+    Repeat: 1
+    InsertBatchSize: 400
+    Batches: 20  # Per Thread
+    SampleSize: 4000
+
+Actors:
+# This insert load phase will run with the default dataset in place, which includes ~7 indexes
+# already.
+- Name: BaselineBestBuyIndexes
+  Type: SamplingLoader
+  Threads: *LoadThreads
+  Phases:
+  - *SamplingLoaderPhaseConfig
+  - {Nop: true}
+  - {Nop: true}
+  - {Nop: true}
+  - {Nop: true}
+
+- Name: DropBaselineIndexes
+  Type: RunCommand
+  Phases:
+  - {Nop: true}
+  - Repeat: 1
+    Database: *db
+    Operations:
+    - OperationMetricsName: DropBaselineIndexes
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: *Collection
+        # This will drop all non-essential indexes (_id and any shard key index remain)
+        index: "*"
+  - {Nop: true}
+  - {Nop: true}
+  - {Nop: true}
+
+- Name: NoIndexes
+  Type: SamplingLoader
+  Threads: *LoadThreads
+  Phases:
+  - {Nop: true}
+  - {Nop: true}
+  - *SamplingLoaderPhaseConfig
+  - {Nop: true}
+  - {Nop: true}
+
+- Name: BuildColumnStoreIndex
+  Type: RunCommand
+  Phases:
+  - {Nop: true}
+  - {Nop: true}
+  - {Nop: true}
+  - Repeat: 1
+    Database: *db
+    Operations:
+    - OperationMetricsName: BuildColumnStoreIndex
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: *Collection
+        indexes:
+          - *ColumnStoreIndexSpec
+  - {Nop: true}
+
+- Name: ColumnStoreIndex
```

Comment:
By now the dataset has gained 120K+ documents compared to the experiment with the default indexes. Do we care?